### PR TITLE
Proposal to initialize Base cWETHv3 market

### PR DESCRIPTION
**Pull request by kevincheng96**
The proposal to initialize the cWETHv3 market.

### CODE REVIEW

File: deployments/base/weth/configuration.json [hunk 1]
```diff
@@ -5,15 +5,15 @@
   "baseTokenAddress": "0x4200000000000000000000000000000000000006",
   "borrowMin": "1e0",
   "pauseGuardian": "0x3cb4653f3b45f448d9100b118b75a1503281d2ee",
-  "storeFrontPriceFactor": 0.5,
+  "storeFrontPriceFactor": 1,
```
Comment (scott-silver) [inline deployments/base/weth/configuration.json:8]: interesting; so the protocol would make nothing for a liquidation?

File: deployments/base/weth/migrations/1689893694_configurate_and_ens.ts [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+        signature: '_setCompSpeeds(address[],uint256[],uint256[])',
+        args: [
+          [cUSDCAddress],
+          [9194444444444444n],
+          [12666666666666667n]
+        ],
+      },
+    ];
+
+    // TODO
```
Comment (scott-silver) [inline deployments/base/weth/migrations/1689893694_configurate_and_ens.ts:135]: don't forget to do this TODO

File: deployments/base/weth/migrations/1689893694_configurate_and_ens.ts [hunk 3]
```diff
@@ -0,0 +1,229 @@
...
+      WETH
+    } = await deploymentManager.getContracts();
+
+    const {
+      comptrollerV2,
+    } = await govDeploymentManager.getContracts();
+
+    // 1.
+    const stateChanges = await diffState(comet, getCometConfig, preMigrationBlockNumber);
+    // TODO: uncomment after contracts are deployed
```
Comment (scott-silver) [inline deployments/base/weth/migrations/1689893694_configurate_and_ens.ts:164]: don't forget to do this TODO

File: deployments/base/weth/configuration.json (hunk 1 above)
Comment (kevincheng96) [reply deployments/base/weth/configuration.json:8]: Yeah, based on the recommendations in this post: https://www.comp.xyz/t/gauntlet-recommendations-ethereum-compound-v3-eth-risk-parameter-ir-curve-incentive-changes-8-4-23/4565

File: deployments/base/weth/migrations/1689893694_configurate_and_ens.ts [hunk 4]
```diff
@@ -0,0 +1,229 @@
...
+        },
+      ],
+      42161: [
+        {
+          baseSymbol: 'USDC',
+          cometAddress: '0xA5EDBDD9646f8dFF606d7448e414884C7d905dCA',
+        }
+      ],
+      8453: [
+        // TODO: uncomment after proposal 171 goes through
```
Comment (scott-silver) [inline deployments/base/weth/migrations/1689893694_configurate_and_ens.ts:211]: one more TODO to do

**Review by scott-silver — APPROVED** [review]
👍

**Comment by kevincheng96** [comment]
Enacted in https://github.com/compound-finance/comet/actions/runs/5837707831/job/15833665695

**Pull request merged**